markhor_flippers/src/markhor_flippers.cpp

```diff
@@ -26,21 +23,21 @@ static int multiplicator = 0;
 void joyCallback(const sensor_msgs::Joy::ConstPtr& joy)
 {
   std_msgs::Float64 msg;
-  if (flipper_mode_front == true)
-  {
-    msg.data = joy->axes[4] * multiplicator;
-    flipper_fl_pub.publish(msg);
-    flipper_fr_pub.publish(msg);
-  }
-  if (flipper_mode_back == true)
+  if (individual_mode == false)
   {
-    msg.data = joy->axes[4] * multiplicator;
-    flipper_rl_pub.publish(msg);
-    flipper_rr_pub.publish(msg);
-  }
-  if (individual_mode == true)
-  {
-
+    if (flipper_mode_fl == true && flipper_mode_fl ==true)
```

Comment:
possible typo

Reply:
Indeed, would have fixed if the line wasn't removed in itself.